stdlib/sql/sqlite.go

```diff
@@ -0,0 +1,169 @@
+package sql
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/execute"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/values"
+)
+
+type SqliteRowReader struct {
+	Cursor      *sql.Rows
+	columns     []interface{}
+	columnTypes []flux.ColType
+	columnNames []string
+	NextFunc    func() bool
```

Comment:
For sqlite3 and that specifically, it would likely be better to just use a memory source and just test using the actual library. The only reason for these in the MySQL one is because we don't want to require a MySQL server be present when running `go test`.

For sqlite, that's not really an issue.